From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Tue, 13 Jan 2026 16:26:29 +0800
Subject: onscreen/native: Reduce the log level of buffer age warnings

They are occuring too often and don't seem represent a bug anywhere.
Just sometimes with triple buffering, Nvidia will reorder its buffers.
This would be expected most when dropping from triple to double buffering
but it seems to also happen even if I force always triple buffering.

Bug: https://gitlab.gnome.org/GNOME/mutter/-/issues/4552
Origin: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/4839
---
 src/backends/native/meta-onscreen-native.c | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/src/backends/native/meta-onscreen-native.c b/src/backends/native/meta-onscreen-native.c
index a95cc72..008afe2 100644
--- a/src/backends/native/meta-onscreen-native.c
+++ b/src/backends/native/meta-onscreen-native.c
@@ -1036,10 +1036,11 @@ get_secondary_gpu_buffer_age (MetaOnscreenNativeSecondaryGpuState *secondary_gpu
 
   if (buffer_age > MAX_SECONDARY_GPU_BUFFER_AGE)
     {
-      g_warning ("Secondary GPU provides buffers of age %i, which is "
-                 "older than supported; ignoring damage rectangles and fully "
-                 "redrawing which may cause increased GPU power consumption",
-                 buffer_age);
+      meta_topic (META_DEBUG_KMS,
+                  "Secondary GPU provides buffers of age %i, which is "
+                  "older than supported; ignoring damage rectangles and fully "
+                  "redrawing which may cause increased GPU power consumption",
+                  buffer_age);
 
       return 0;
     }
